llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2951,6 +2960,50 @@ class VPBasicBlock : public VPBlockBase {
   BasicBlock *createEmptyBasicBlock(VPTransformState::CFGState &CFG);
 };
 
+/// A special type of VPBasicBlock that wraps an existing IR basic block.
+/// Recipes of the block get added before the terminator of the wrapped IR basic
+/// block.
```

Comment:
Yes, will update https://github.com/llvm/llvm-project/pull/92651 soon based on this PR. Left the order as is for now, to avoid updating IR tests. Can adjust order as follow-up